third_party/CMakeLists.txt

```diff
@@ -26,6 +26,12 @@
 #  POSSIBILITY OF SUCH DAMAGE.
 #
 
-if(OT_BUILTIN_MBEDTLS)
+if(NOT OT_EXTERNAL_MBEDTLS)
     add_subdirectory(mbedtls)
 endif()
+
+add_library(openthread-mbedtls INTERFACE)
+
+target_link_libraries(openthread-mbedtls
+    INTERFACE $<IF:$<BOOL:${OT_EXTERNAL_MBEDTLS}>,${OT_EXTERNAL_MBEDTLS}, mbedtls>
```

Comment:
new line for the expression $<IF:$<BOOL:${OT_EXTERNAL_MBEDTLS}>,${OT_EXTERNAL_MBEDTLS}, mbedtls>